v1/plugin/flags.go

```diff
@@ -0,0 +1,54 @@
+package plugin
+
+import "github.com/urfave/cli"
+import "time"
+
+var (
+	flConfig = cli.StringFlag{
+		Name:  "config",
+		Usage: "config file to use",
+	}
+	// If no port was provided we let the OS select a port for us.
+	// This is safe as address is returned in the Response and keep
+	// alive prevents unattended plugins.
+	flPort = cli.StringFlag{
+		Name:  "port",
+		Usage: "port to listen on",
+	}
+	// If PingTimeoutDuration was provided we set it
+	flLogLevel = cli.IntFlag{
+		Name:  "log-level",
+		Usage: "log level - 0:panic 1:fatal 2:error 3:warn 4:info 5:debug",
```

Comment:
Consider keeping the same format as snapteld has: ` 1-5 (Debug, Info, Warning, Error, Fatal; default: 3)`.